backend/internal/entity/content.go

```diff
@@ -0,0 +1,11 @@
+package entity
+
+import "gorm.io/gorm"
+
+type Content struct {
+	gorm.Model
+	Title string `gorm:"size:255;not null;" json:"title"`
+	Path string `gorm:"size:255;not null;" json:"path"`
+	ModuleID uint 
+	Module Module `gorm:"foreignKey:ModuleID;references:ID" json:"-"`
```

Comment:
I am no sure I understand the logic behind this field. Where would the content (video, text, images, etc) will be stored?

Reply:
This comment is kinda outdated based on our conversation on Discord.